### DISCUSSION TITLE
Tailwind in a template

### DISCUSSION THREAD

**Discussion by janpelc** [typemill/typemill#476] [Q&A] [answered]
Hi, I have a question about using Tailwind in a template. I know that it is used in core Typemill. The Tailwind documentation describes how to use it without NPM. But I was not successful when I tried to use it in a template. Will there be a procedure about this in future documentation?  Alternatively, I would be glad for advice on how to make it work in the template. Thanks for the answer

**Comment by trendschau** [comment, marked as answer]
Hi Jan,

do you want to use Tailwind in a plugin-template or in a theme? 

Typemill uses an npm-generated css-file with Tailwind, so there are only some specific styles. If styles are missing that you want to use in a plugin-template, then my workaround is to include another css file that adds those styles. See here: https://typemill.net/plugin-developers/plugin-methods/addcss

If you create a theme, then the existing css-file in Typemill is probably useless and you have to generate and include your own Tailwind css like described here: https://typemill.net/theme-developers/theme-variables/baseurl

Not super sure if I misunderstand the problem. If the documentation is unclear, please let me know what is missing and I will try to add or rewrite.

Just a general recommendation: I still use Tachyons framework for themes, because it is much easier without all these npm-stuff, and it follows the same philosophy of utility-css.

**Reply by janpelc** [reply]
Thank you for your prompt reply. Yes, I meant Tailwind when used in a theme. The second link might help, thank you. I'll try the Tachyons. I just didn't feel like learning another technology. But simplicity is a big benefit. That's why I liked Typemill. Or HTMX. Thanks for a great job with Typemill and I look forward to more features.
